### PR TITLE
Vet clinic database: create animals table

### PR DESCRIPTION
**Pull request by kinginthenorthcodez**
## Vet clinic database: create animals table
- Create tables in SQL 👍🏽 
- Insert and query data in SQL. 👍🏽 

### Following are snapshots of related queries ⏬ 
- query ```CREATE TABLE animals(
id INT,
name TEXT,
date_of_birth DATE,
escape_attempts INT,
neutered BOOLEAN,
weight_kg DECIMAL
);```
- query ``` INSERT INTO animals(id,name,date_of_birth,escape_attempts,neutered,weight_kg) values(1,'Agumon','2020-02-03
',0,false,10.23),(2,'Gabumon','2018-11-15',2,true,8),(3,'Pikachu','2021-01-07',1,false,15.04),(4,'Devimon','2017-05-12',
5,true,11); ```
- 
image

- query ```SELECT * FROM animals WHERE name LIKE '%mon';```
- 
image

- query ```SELECT name FROM animals WHERE date_of_birth  BETWEEN '2016-01-01' AND '2019-12-01'
;```
image

- querry ```SELECT name FROM animals WHERE neutered=true AND escape_attempts < 3;```
- 
image

- querry ```SELECT date_of_birth FROM animals WHERE name='Agumon' OR name='Pikachu';```
- 
image

- querry ```SELECT name,escape_attempts FROM animals WHERE weight_kg > 10.5;```
- 
image

- querry ```SELECT * FROM animals WHERE neutered = true;```
- 
image

- querry ```SELECT * FROM animals WHERE NOT name =' Gabumon';```
- 
image

- querry ```SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;```
- 
image

Warm Regards
@kinginthenorthcodez

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE TABLE animals(
+id INT,
+name TEXT,
```
Comment (sja-thedude) [inline schema.sql:5]: - [x] Kindly make sure that name is of varchar data type.

File: data.sql [hunk 2]
```diff
@@ -3,3 +3,6 @@
 INSERT INTO animals (name) VALUES ('Luna');
 INSERT INTO animals (name) VALUES ('Daisy');
 INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals(id,name,date_of_birth,escape_attempts,neutered,weight_kg) values(1,'Agumon','2020-02-03
+',0,false,10.23),(2,'Gabumon','2018-11-15',2,true,8),(3,'Pikachu','2021-01-07',1,false,15.04),(4,'Devimon','2017-05-12',
```
Comment (sja-thedude) [inline data.sql:7]: - [x] Kindly make sure that neutered is true for Agumon as per the requirements.

File: queries.sql [hunk 3]
```diff
@@ -1,3 +1,13 @@
 /*Queries that provide answers to the questions from all projects.*/
 
 SELECT * from animals WHERE name = 'Luna';
+
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth  BETWEEN '2016-01-01' AND '2019-12-01'
```
Comment (sja-thedude) [inline queries.sql]: - [x] Kindly make sure that the end date should be '2019/31/12' as per the requirements.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @kinginthenorthcodez, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Professional Readme
✔️ Correct git-flow

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 2 above)
Comment (kinginthenorthcodez) [reply data.sql:7]: - @sja-thedude  Thank you noticing that , suggested changes made 👍🏽 
image

File: schema.sql (hunk 1 above)
Comment (kinginthenorthcodez) [reply schema.sql:5]: - suggested changes made 👍🏽 
image

File: queries.sql (hunk 3 above)
Comment (kinginthenorthcodez) [reply queries.sql]: @sja-thedude 
- Suggested changes made 👍🏽

**Review by keroloussamy — APPROVED** [review]
Hi @kinginthenorthcodez, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap

## Optional suggestions
_Check the comments under the review._

- [ ] x

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kinginthenorthcodez** [comment]
Merged #1 inot development

**Pull request merged**